README.rst

```diff
@@ -75,11 +75,11 @@ Supported Spark Versions
 
 At the moment we support:
 
-+-------------------------------------------+
-| sparkly 2.x | Spark 2.0.x and Spark 2.1.x |
-+-------------------------------------------+
-| sparkly 1.x | Spark 1.6.x                 |
-+-------------------------------------------+
++---------------------------------------------------------------------------+
+| sparkly 2.x | Spark 2.0.x and Spark 2.1.x and Spark 2.2.x and Spark 2.4.x |
```

Comment:
if we are not sure that spark 2.4.x and spark 2.2/1.x are compatible, let's have a separate row here like sparkly >= 2.7 is spark 2.4 compatible